Dockerfile

```diff
@@ -1,26 +1,23 @@
-FROM node:20-alpine
+# Multi-platform Node 20 Alpine
+FROM --platform=$BUILDPLATFORM node:20-alpine
```

Comment:
This may be needed for cross compilation support, but we can build multi-platform images using [multiple native nodes](https://docs.docker.com/build/building/multi-platform/#multiple-native-nodes). This is what happens on our CI.

In the `hyperswitch` repo, we don't read the `$BUILDPLATFORM` build argument at all:
https://github.com/juspay/hyperswitch/blob/3f18c944117007528093799ffcd779c6d31839f9/Dockerfile